dvc/remote/s3.py

```diff
@@ -234,3 +249,14 @@ def _generate_download_url(self, path_info, expires=3600):
         return self.s3.generate_presigned_url(
             ClientMethod="get_object", Params=params, ExpiresIn=int(expires)
         )
+
+    def _collect_dir(self, path_info):
+        root = path_info.path
+
+        return [
+            {
+                self.PARAM_CHECKSUM: entry["ETag"],
+                self.PARAM_RELPATH: os.path.relpath(entry["Key"], start=root),
+            }
+            for entry in self._list_objects(path_info)
```

Comment:
This is called `item` in other places and `entry` here.

Reply:
@Suor , will address this on this one #2648